docs/modules/ROOT/partials/cmos-health-checks.adoc

```diff
@@ -158,6 +158,19 @@ Dropped ticks are usually a sign of resource contention, specifically CPU conten
 
 *Further Reading*: xref:7.0@server:install:sizing-general[Sizing Guidelines]
 
+[#CB90067]
+=== Service Rollback to Zero (CB90067)
+
+*Background*: A service has been forced to roll back to zero.
```

Comment:
Roll what back? Should we mention DCP explicitly because "rollback" on its own is ambiguous?

Reply:
Fixed :)